java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -184,6 +185,7 @@ private static Object[] stringCommands(BaseTransaction<?> transaction) {
 
         return new Object[] {
             OK, // set(stringKey1, value1)
+            stringKey1, // randomKey()
```

Comment:
this could be flaky. Since we're using this database for all tests, we could get in the situation where there's more than one key, and this returns something else.  In which case, the test would fail. 

Can you flush the database before the set, to guarantee that the database is empty...?  I think there might be a chance that another test adds another key while the transaction is processing, but I would think that would be very rare.

Reply:
- [ ] harden test